docs/frameworks/sveltekit.md

```diff
@@ -0,0 +1,229 @@
+---
+title: SvelteKit | Frameworks
+---
+
+# SvelteKit
+
+> For `Type declarations`, `Prompt for update` and `Periodic SW Updates` go to [Svelte](/frameworks/svelte.html) entry.
+> 
+
+You should remove all references to [SvelteKit service worker module](https://kit.svelte.dev/docs#modules-$service-worker) <outbound-link />
+to disable it on your application.
+
+Since `SvelteKit` will use `SSR / SSG`, we need to add the `ReloadPrompt` component using `dynamic import`. `Vite Plugin PWA`
```

Comment:
I'd be curious for more details here as to why SSR / SSG specifically requires the reload prompt. Wouldn't all sites require the reload prompt?

A slightly more natural way to word this would be:
```suggestion
Since `SvelteKit` uses `SSR / SSG`, we need to add the `ReloadPrompt` component using `dynamic import`. `Vite Plugin PWA`
```